packages/next/taskfile.js

```diff
@@ -1358,7 +1358,14 @@ externals['native-url'] = 'next/dist/compiled/native-url'
 export async function ncc_native_url(task, opts) {
   await task
     .source(relative(__dirname, require.resolve('native-url')))
-    .ncc({ packageName: 'native-url', externals, target: 'es5' })
+    .ncc({
+      packageName: 'native-url',
+      externals: {
+        ...externals,
+        querystring: 'next/dist/compiled/querystring-es3',
+      },
+      target: 'es5',
```

Comment:
Might be a future PR: but can we bump the target now?

Reply:
Yeah we can probably remove these and use the default in a follow-up